src/SFA.DAS.PSRService.Web/Controllers/ReportController.cs

```diff
@@ -278,6 +282,29 @@ public IActionResult Change(OrganisationViewModel organisationVm)
             return new RedirectResult(Url.Action("Edit", "Report"));
         }
 
+
+        [Route("Amend")]
+        [Authorize(Policy = PolicyNames.CanEditReport)]
+        public IActionResult Amend()
+        {
+            var report = _reportService.GetReport(_currentPeriod.PeriodString, EmployerAccount.AccountId);
```

Comment:
I wouldn't have pulled the report back from the service and passed that to the command handler, I'd have passed the input parameters (period and account id) to the handler and let it get the report.